### PR TITLE
feat: message banners table migration

### PR DESCRIPTION
**Pull request by nunogois**
https://linear.app/unleash/issue/2-1485/db-create-migration-for-a-new-internal-message-banners-table

Adds a DB migration for a new `message_banners` table.

### CODE REVIEW

File: src/migrations/20231012082537-message-banners.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+exports.up = function (db, cb) {
+    db.runSql(
+        `
+        CREATE TABLE IF NOT EXISTS message_banners
+            (
+                id SERIAL PRIMARY KEY NOT NULL,
+                enabled BOOLEAN DEFAULT true NOT NULL,
+                message TEXT NOT NULL,
+                variant TEXT,
```
Comment (daveleek) [inline src/migrations/20231012082537-message-banners.js:11]: Is this meant as type of message? Like warning, info, error etc?

**Review by daveleek — APPROVED** [review]
LGTM!

File: src/migrations/20231012082537-message-banners.js (hunk 1 above)
Comment (nunogois) [reply src/migrations/20231012082537-message-banners.js:11]: Yep, exactly! Do you think we should have a different type here?

Comment (daveleek) [reply src/migrations/20231012082537-message-banners.js:11]: I was just pondering the use case, as the name was so close to a part of our domain (variants and features/toggles). We could call it message_type or banner_type? Just suggestions though

Comment (nunogois) [reply src/migrations/20231012082537-message-banners.js:11]: That's a fair point 🤔 I think I prefer to proceed like this to ensure existing external message banner payloads don't break, but it's something I'll take into account and get back to if we identify some confusion.

**Pull request merged**